server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -183,6 +191,45 @@ public Invocations getInvocations() {
       return CompletableFuture.completedFuture(responseBuilder.build());
    }
 
+   private CompletionStage<RestResponse> getAllDetails(RestRequest request) {
+      NettyRestResponse.Builder responseBuilder = new NettyRestResponse.Builder();
+      String cacheName = request.variables().get("cacheName");
+
+      Cache<?, ?> cache = invocationHelper.getRestCacheManager().getCache(cacheName, APPLICATION_JSON, APPLICATION_JSON, request);
```

Comment:
Why APPLICATION_JSON here? My understanding is that this method won't read cache data, so no need to decorate the cache with JSON.